package.json

```diff
@@ -46,6 +46,9 @@
     },
     "appImage": {
       "systemIntegration": "doNotAsk"
+    },
+    "dmg": {
```

Comment:
For future organization I think we should put these under their platform headings, but that we can totally do at another time.

Reply:
@marcparadise and I interpreted the following from [electron-builder's documentation on DMG](https://www.electron.build/configuration/dmg) as requiring `dmg` to be at this level in the `build` portion of the data structure:

> The top-level dmg key contains set of options instructing electron-builder on how it should build DMG.